docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -91,3 +91,7 @@ Apart from `extra_links.tmpl` and `extra_tabs.tmpl`, there are other useful temp
 ## Customizing gitignores, labels, licenses, locales, and readmes.
 
 Place custom files in corresponding sub-folder under `custom/options`.
+
+## Customizing the look of Gitea
+
+Gitea has two builtin themes, the default theme `gitea`, and a dark theme `arc-green`. To change the look of your Gitea install change the value of `[ui] DEFAULT_THEME` in `app.ini` to another one of the available options.
```

Comment:
`built-in` instead of `builtin`.
`[UI] DEFAULT_THEME` might be a little bit confusing. I would propose to create a link to the `UI` section of the CSC (https://docs.gitea.io/en-us/config-cheat-sheet/#ui-ui).

Reply:
Thanks @JonasFranzDEV 😄 

I've made these changes.